jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildImageTask.java

```diff
@@ -174,4 +179,42 @@ BuildImageTask setJibExtension(JibExtension jibExtension) {
     this.jibExtension = jibExtension;
     return this;
   }
+
+  /**
+   * Converts the {@link ImageParameters} to an {@link Authorization}.
```

Comment:
I don't see `ImageParameters` in the method signature. `AuthParameters`?

Reply:
Oops yeah, outdated comments.